compiler/src/iree/compiler/Dialect/HAL/Transforms/DumpExecutableBenchmarks.cpp

```diff
@@ -77,7 +79,8 @@ static DispatchParamsMap gatherDispatchParams(mlir::ModuleOp moduleOp) {
       for (auto workloadValue : workloadValues) {
         APInt workloadConstValue;
         if (!matchPattern(workloadValue, m_ConstantInt(&workloadConstValue))) {
-          // Non-constant workload; skip this dispatch.
+          LLVM_DEBUG(llvm::dbgs()
+                     << "Skipping dispatch (non-constant workload)\n");
```

Comment:
I tried to add context to these, but the full op is pretty verbose. Worth including?

```mlir
stream.cmd.dispatch @add_one_dispatch_0::@embedded_elf_x86_64::@add_one_dispatch_0_generic_D_i32[%0](%5, %7 : i32, i32) {
  ro %arg1[%c0 for %1] : !stream.resource<external>{%1},
  wo %arg2[%c0 for %1] : !stream.resource<external>{%1}
} attributes {hal.interface.bindings = [#hal.interface.binding<0, 0>, #hal.interface.binding<0, 1>]}
```

Reply:
you could print the symbol name referenced ("skipping benchmark generation for dispatch to @foo (non-constant workload)") so that people can find it

Reply:
Looks like `stream.cmd.dispatch` can have multiple entry points? I printed the first one.

https://github.com/openxla/iree/blob/caaf7ddcabcf2098031ce9009e39b9142410c456/compiler/src/iree/compiler/Dialect/Stream/IR/StreamOps.td#L2731-L2748